features/step_definitions/basic_steps.rb

```diff
@@ -31,12 +31,18 @@
 def page_path(path)
   if path == 'Create Article'
     new_article_path
-  elsif path == 'Holger is the best'
-    article_title = Article.find_by(title: path)
-    article_path(article_title)
   elsif path == 'Index'
     root_path
+  elsif (path == 'Holger is the best') || (path == 'Jade loves cookies')
+    find_acticle(path)
+  elsif (path == 'Jade loves SailorCookie') # || (add new path here)
+    find_acticle(path)
   else
-    raise "You need to add #{path} to page_path"
+    raise "You need to add #{path} to page_path function"
   end
 end
+
+def find_acticle(path)
+  article_title = Article.find_by(title: path)
```

Comment:
And at this point, you are not getting the `article_title` but rather the entire object. So IMO this variable should be named `article`